libraries/standard/mqtt/cbmc/proofs/MQTT_DeserializeAck/Makefile

```diff
@@ -0,0 +1,17 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Apache??

Reply:
This is auto copied from the CBMC templates.